Index: src/App.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/App.vue b/src/App.vue
--- a/src/App.vue	(date 1655810575224)
+++ b/src/App.vue	(date 1655810575224)
@@ -29,7 +29,7 @@
   },
   methods: {
     updateFilters(filter) {
-      console.log(this.catalog.filter(product => product.marketType === filter))
+      this.catalog.filter(product => product.marketType === filter)
       if(!this.filters.includes(filter)) {
         this.filters.push(filter)
       } else if(this.filters.indexOf(filter) != -1) {
